packages/core/src/components/toast/toaster.tsx

```diff
@@ -89,6 +93,9 @@ export class Toaster extends AbstractComponent<IToasterProps, IToasterState> imp
      * The `Toaster` will be rendered into a new element appended to the given container.
      */
     public static create(props?: IToasterProps, container = document.body): IToaster {
+        if (props != null && props.inline != null) {
```

Comment:
would prefer explicit checks against `undefined`; we don't really need to support nulls